tests/test_tpa_pipeline.py

```diff
@@ -127,75 +127,37 @@ def test_get_consent_status(self):
             consent.delete()
             ec_user.delete()
 
-    def test_verify_data_sharing_consent(self):
+    def test_handle_enterprise_logistration(self):
         """
-        Test that we correctly verify consent status.
+        Test that we correctly handle incoming enterprise logistration requests.
```

Comment:
This docstring isn't really helpful -- it basically just restates the name of the test.